ui/README.md

```diff
@@ -22,11 +21,13 @@ $ yarn
 
 ## Running / Development
 
-First, make sure nomad is running. The UI, in development mode, runs independently from Nomad, so this could be an official release or a dev branch. Likewise, Nomad can be running in server mode or dev mode. As long as the API is accessible, the UI will work as expected.
+UI in development mode defaults to using fake generated data, but you can configure it to proxy a live running nomad process by setting `USE_MIRAGE` environment variable to `false`.  First, make sure nomad is running. The UI, in development mode, runs independently from Nomad, so this could be an official release or a dev branch. Likewise, Nomad can be running in server mode or dev mode. As long as the API is accessible, the UI will work as expected.
 
-* `ember serve`
+* `USE_MIRAGE=false ember serve`
 * Visit your app at [http://localhost:4200](http://localhost:4200).
 
+You may need to reference the direct path to `ember`, typically in `./node_modules/.bin/ember`.
```

Comment:
I didn't install ember-cli directly?  Is there a concern about using a globally installed ember package that doesn't match what's set in yarn config?

Reply:
It’s fine to use a different version globally, which is maybe a bit confusing without context. But maybe we could bypass that question altogether by suggesting `yarn start`, which uses the Ember CLI version installed in `node_modules`? It’s guaranteed to work if `yarn` has been run, but it’s also not conventional 🤔